apps/src/bin/quiche-server.rs

```diff
@@ -690,7 +799,10 @@ fn validate_token<'a>(
 
 fn handle_path_events(client: &mut Client) {
     while let Some(qe) = client.conn.path_event_next() {
+        trace!("There was an event");
```

Comment:
Seems not useful in it's current form :smile:

Reply:
lol true, removed 👍